File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/vo/dept/DeptDetailVO.java
package ink.ckx.mo.system.api.model.vo.dept;

import ink.ckx.mo.common.web.enums.StatusEnum;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

@Schema(description = "部门详情VO")
@Data
public class DeptDetailVO {

    @Schema(description = "部门ID")
    private Long id;

    @Schema(description = "部门名称")
    private String name;

    @Schema(description = "父部门ID")
    private Long parentId;

    @Schema(description = "状态")
    private StatusEnum status;

    @Schema(description = "排序")
    private Integer sort;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/vo/dept/DeptDetailVO.kt
package ink.ckx.mo.admin.api.model.vo.dept

import ink.ckx.mo.common.web.enums.StatusEnum
import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "部门详情VO")
data class DeptDetailVO(

    @Schema(description = "部门ID")
    var id: Long,

    @Schema(description = "部门名称")
    var name: String,

    @Schema(description = "父部门ID")
    var parentId: Long,

    @Schema(description = "状态")
    var status: StatusEnum,

    @Schema(description = "排序")
    var sort: Int,
) 